### DISCUSSION TITLE
[Feature Request] Need Uninstall

### DISCUSSION THREAD

**Discussion by ahdung** [ventoy/Ventoy#1910] [Ideas]
Should provide a way to recovery user's usb disk.

**Comment by ventoy** [comment]
Option ----> Clear Ventoy

![image](https://user-images.githubusercontent.com/59477474/195280379-b284a042-4cc5-4bc3-9855-e6296b92c8cf.png)

**Reply by ahdung** [reply]
Thanks, please add uninstallation knowledge to document and FAQ.
